tools/dashboards/README.md

```diff
@@ -0,0 +1,27 @@
+# Documentation
+
+This documentation provides instructions for setting up grafana dashboards to see metrics emitted from the inference extension and model servers.
+
+## Requirements
+
+Please follow [metrics](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/pkg/ext-proc/metrics) page to configure the proxy to enable all metrics.
+
+## Load Inference Extension dashboard into Grafana
+
+Please follow [grafana instructions](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/import-dashboards/) to load the dashboard json.
+
+## Configure Google Managed Prometheus as source for metrics
+
+If you run the inferece gateway with [Google Managed Prometheus], please follow the [instructions](https://cloud.google.com/stackdriver/docs/managed-prometheus/query) to configure Google Managed Prometheus as data source for the grafana dashboard.
```

Comment:
are the brackets around [Google Managed Prometheus] intentional?

Reply:
Added the link.